### DISCUSSION TITLE
Maldita Castilla Port issues

### DISCUSSION THREAD

**Discussion by FoxSoratto** [rg35xx-cfw/rg35xx-cfw.github.io#225] [Q&A] [answered]
Guys, i've tried my best to make it work. Has anyone managed to get it running? I have the game files from the original MC free download, idk if im having issues with the .sh archive (the only source that gives some hint is the Batocera wiki). Any help will be MASSIVELY appreciated.

Besides that, there's any know repo with all working ports for this particular version and their respective .sh? If not, there's the ideia :)

**Comment by acmeplus** [comment, marked as answer]
Ports are going to be consolidated within PortMaster, so they are actively maintained by that time. There will be updates once the first version of [Knulli](https://knulli.org) comes out.

**Reply by FoxSoratto** [reply]
Got It! 

My dumb ass totally ignored the fact that after flashing v40, the MC .sh file comes with It and ready to run It after putting all MC archives in the right folder. Reflashing did the trick. Thx!
